prospector/client/cli/prospector_client.py

```diff
@@ -230,7 +223,7 @@ def prospector(  # noqa: C901
                 preprocessed_commits, advisory_record, rules=rules
             )
 
-            annotated_candidates = rank(annotated_candidates, model_name=model_name)
```

Comment:
I'd rather have a wrapper function `rank` (that could be implemented as a simple sort, but it does not have to) to make the "ranking" operation more explicit in the code structure